src/Symfony/Component/DependencyInjection/Loader/IniFileLoader.php

```diff
@@ -49,4 +50,32 @@ public function supports($resource, $type = null)
     {
         return is_string($resource) && 'ini' === pathinfo($resource, PATHINFO_EXTENSION);
     }
+
+    /**
+     * Note that the following features are not supported and are kept as is:
+     *  * strings with escaped quotes are not supported "foo\"bar"
+     *  * string concatenation ("foo" "bar")
+     */
+    private function phpize($value)
+    {
+        // trim on the right as a comment leaves the whitespaces in place
+        $value = rtrim($value);
+
+        switch (true) {
+            case defined($value):
+                return constant($value);
+            case 'yes' === $value || 'on' === $value:
+                return true;
+            case 'no' === $value || 'off' === $value || 'none' === $value:
+                return false;
+            case isset($value[0]) && (
+                ("'" === $value[0] && "'" === $value[strlen($value) - 1]) ||
+                ('"' === $value[0] && '"' === $value[strlen($value) - 1])
+            ):
+                // quoted string
```

Comment:
It was fatal before :/ https://3v4l.org/VgDvJ

With SCANNER_RAW it parses though.. but that doesnt mean it's valid, it's raw :)